### PR TITLE
Added option to access spFlat file

### PR DESCRIPTION
**Pull request by belaa**
This PR modifies spec.py to allow the option to access data from spFlat files for individual exposures.

### CODE REVIEW

File: bossdata/spec.py [hunk 1]
```diff
@@ -518,7 +522,6 @@ def get_valid_data(self, exposure_index=None, camera=None, pixel_quality_mask=No
             first_index = float(get_fiducial_pixel_index(10.0**hdu['loglam'][0]))
             if abs(first_index - round(first_index)) > 0.01:
                 raise RuntimeError('Wavelength grid not aligned with fiducial grid.')
-            first_index = int(round(first_index))
```
Comment (dkirkby) [inline bossdata/spec.py]: Why did you remove this line? It looks like first_index needs to be an int before it used to create a slice.

Comment (belaa) [reply bossdata/spec.py]: @dkirkby I think I was testing something and removed it by mistake. Should I add it back in and submit a new PR?

Comment (dkirkby) [reply bossdata/spec.py]: You don't need a new PR.  Just commit the fix to the same branch and push.  It should show up here automagically.

**Comment by dkirkby** [comment]
This branch has been merged into #125.
